### PR TITLE
Upate, delete and transaction

### PR DESCRIPTION
**Pull request by Fayob**
- Add column species of type string to animals table.
- Insert the following data
- Add these insertions to your data.sql file
- Add transaction
- Add update and delete from table

Screen Shot 2022-12-06 at 8 41 22 PM
Screen Shot 2022-12-06 at 9 05 00 PM
Screen Shot 2022-12-06 at 9 52 43 PM
Screen Shot 2022-12-06 at 10 41 27 PM
Screen Shot 2022-12-06 at 10 42 00 PM
Screen Shot 2022-12-06 at 10 42 25 PM

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,64 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg between 10.4 and 17.3;
+
+--  Transaction
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified' WHERE neutered = false;
+
+ROLLBACK;
```
Comment (mikethreels) [inline queries.sql]: - [ ] in this transaction you don't need to specify if the animal was neutered or not
- [ ] please make sure the verify if the changes where made after the update and if the changes were undone after the rollback

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Favour, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Fayob) [reply queries.sql]: @mikethreels, I verified the changes before roll back but I don't know if there is another way you want me to do it, It'll be helpful if you can be more explicit with what you want me to do. Thanks

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Fayob 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**